odl/tomo/analytic/filtered_back_projection.py

```diff
@@ -109,12 +265,20 @@ def fft_filter(x):
             scale = (ray_trafo.geometry.src_radius /
                      (ray_trafo.geometry.src_radius +
                       ray_trafo.geometry.det_radius))
+
+            if ray_trafo.geometry.pitch != 0:
+                # In helical each projection hits the detector less than once.
```

Comment:
Unclear what this means. In my world view, a projection is exactly what the detector spits out for a certain angle or whatever parametrizes the geometry.